FFXIVClientStructs/FFXIV/Client/Game/MJI/MJIPastureHandler.cs

```diff
@@ -53,7 +78,39 @@ public unsafe struct MJIAnimal {
     /// The animal's current mood. Ranges from 0 for "very unhappy" (lightning icon) to 4 for "very happy" (sun icon).
     /// </summary>
     [FieldOffset(0x26)] public byte Mood;
-    [FieldOffset(0x27)] public ushort Leavings; // ?? unsure why this is a ushort.
+
+    [FieldOffset(0x27), Obsolete("Use ManualLeavingsAvailable & UnderCare fields instead.")] public ushort Leavings; // ?? unsure why this is a ushort.
+
+    /// <summary>
+    /// True if there are some leavings that you can manually collect.
+    /// </summary>
+    [FieldOffset(0x27)] public bool ManualLeavingsAvailable;
+
+    /// <summary>
+    /// True if this animal is managed by a mammet.
+    /// </summary>
+    [FieldOffset(0x28)] public bool UnderCare;
+
+    /// <summary>
+    /// True if mammet was paid for care already - if true, dismissing and re-entrusting won't cost anything.
+    /// </summary>
+    [FieldOffset(0x29)] public bool WasUnderCare;
+
+    /// <summary>
+    /// True if not cared for, either because mammet was not assigned, or because it was not paid.
+    /// </summary>
+    [FieldOffset(0x2A)] public bool CareHalted;
+
+    /// <summary>
+    /// 0 if not under care, otherwise Item row id of used food.
+    /// </summary>
+    [FieldOffset(0x2A)] public uint AutoFoodItemId;
```

Comment:
I just glanced over this and found this to have the same offset as `CareHalted` above it. Mistake?

Reply:
Yep, good catch, thanks! Fixed.